src/test/kotlin/no/nav/syfo/api/endpoints/VedtakEndpointsSpek.kt

```diff
@@ -196,6 +201,22 @@ object VedtakEndpointsSpek : Spek({
                             pVedtak.uuid shouldBeEqualTo vedtakResponse.uuid
                         }
                     }
+                    it("Sets vedtak ferdigbehandlet and updates veileder") {
+                        val vedtak = createVedtak(vedtakRequestDTO)
+                        vedtak.ferdigbehandletAt shouldBe null
+                        val path = urlVedtakFerdigbehandling.replace("{$vedtakUUIDParam}", vedtak.uuid.toString())
```

Comment:
Kan man ikke legge til en path param i requesten, i stedet for å drive med string replace?

Reply:
Ja, skal forsøke å forbedre.